plugins/underflip/resorts/updates/create_resorts_table.php

```diff
@@ -12,6 +12,7 @@ public function up()
             $table->increments('id');
             $table->string('title');
             $table->string('url_segment')->unique();
+            $table->string('booking_affiliate_url');
```

Comment:
```suggestion
            $table->text('booking_affiliate_url');
```

Probably a good idea to give some extra ceiling room in case there's a very long affiliate URL (to accommodate an unknown amount of affiliate URL query params)

Reply:
Smart... :)